respy/tests/codes/auxiliary.py

```diff
@@ -112,14 +101,22 @@ def compare_est_log(base_est_log):
         else:
 
             is_floats = False
-            try:
-                int(shlex.split(alt_line)[0])
-                is_floats = True
-            except ValueError:
-                pass
-            # We need to cut the floats some slack. It might very well happen that in the
-            # very last digits they are in fact different across the versions.
+            entries = shlex.split(alt_line)
+            if len(entries) >= 2:
+                try:
+                    int(entries[1])
+                except ValueError:
+                    is_floats = True
+
+            else:
+                try:
+                    int(entries[0])
+                except ValueError:
+                    is_floats = True
+
             if not is_floats:
+                print(alt_line)
+                print(base_line)
```

Comment:
Are these leftovers?

Reply:
Yes, I removed it. Also, the whole function gets kicked out soon, when you remove the logging.